active/af1-identify-phi.py

```diff
@@ -60,9 +63,9 @@ def generate_job_id(**kwargs):
               " {} total notes scheduled".format(row[0], row[1], (common_variables.MAX_BATCH_SIZE - total_job_count)))
         total_job_count -= row[1]
         if total_job_count <= 0:
-            print("Job Batch for hdcpupdatedate: {}  contains {} notes"
+            print("Job Batch for hdcpupdatedate: {}  contains 0 notes"
                   " and exceeds cumulative total per-run Job Size of {}."
-                  " Breaking early.".format(row[0], row[1], common_variables.MAX_BATCH_SIZE))
+                  " Breaking early.".format(row[0], common_variables.MAX_BATCH_SIZE))
```

Comment:
i thought the row[1] error is in populate_blobid_in_job_table() L107 (https://github.com/FredHutch/nlp-airflow-dags/blob/33d289245d7d3cee30c37638e4408c7f1b6854e9/active/af1-identify-phi.py#L107), not here.